### PR TITLE
plan

### PR DESCRIPTION
**Pull request by jackharrhy**
feel free to comment on plan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+- use go
+  - quick to write in
+  - folks in fogo.sh have go exp.
+  - as much as rust is cool, we want to move fas for this
+
+- for mvp
+  - have a file in repo that'll contain a list of 'actions'
+    - file format can be toml, easy to read and write
+      - use https://github.com/BurntSushi/toml
+  - reading this file, and playing the actions, you can bascially reach the current 'state' of the library
```
Comment (nint8835) [inline README.md]:
```suggestion
  - reading this file, and playing the actions, you can basically reach the current 'state' of the library
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,21 @@
 # the fogo.sh distributed-library
 
-_wip_
+your friends have books, therefore you do too books
```
Comment (nint8835) [inline README.md]:
```suggestion
your friends have books, therefore you do too
```

**Review by nint8835 — CHANGES_REQUESTED** [review]

**Comment by mwln** [comment]
what would be the expected state for two people adding books same time? how will the "db" be synced across instances of the library

**Comment by ranguli** [comment]
finish your other projects first :(

**Comment by jackharrhy** [comment]
no more go

**Pull request closed without merging**